l10n_es_vat_book/report/vat_book_xlsx.py

```diff
@@ -275,31 +275,20 @@ def fill_table_issued(lines, vatbook=False):
                     # Clave de operación
                     col += 1
                     # Total factura
-                    tax1 = lin.mapped("tax_line_ids.tax_id").filtered(
-                        lambda r: r.id not in taxes + taxes2
-                    ).ids
-                    tax1 = lin.mapped("tax_line_ids").filtered(
-                        lambda r: r.tax_id.id in tax1
-                    )
-                    tax2 = lin.mapped("tax_line_ids.tax_id").filtered(
-                        lambda r: r.id in taxes
-                    ).ids
-                    tax2 = lin.mapped("tax_line_ids").filtered(
-                        lambda r: r.tax_id.id in tax2
-                    )
-                    total = sum(x.total_amount for x in tax1)
-                    total += sum(x.tax_amount for x in tax2)
+                    base_imponible = sum(x.base_amount for x in lines_all)
```

Comment:
Hay que utilizar las ventajas que da el ORM y siempre nombres de variables en inglés (aplica a todo el PR)

```suggestion
                    subjected_base = sum(lines_all.mapped('base_amount'))
```